### PR TITLE
Add CupertinoScrollbar to the catalog

### PR DESCRIPTION
**Pull request by xster**

### CODE REVIEW

File: src/_data/catalog/widgets.json [hunk 1]
```diff
@@ -586,6 +586,16 @@
     "link": "https://docs.flutter.io/flutter/cupertino/CupertinoPopupSurface-class.html",
     "image": "<img alt='' src='/images/catalog-widget-placeholder.png'>"
   },
+  {
+    "name": "CupertinoScrollbar",
+    "description": "An iOS-style scrollbar that appears while scrolling lists",
```
Comment (RedBrogdon) [inline src/_data/catalog/widgets.json]: What about "An iOS-style scrollbar that indicates which portion of a Scrollable widget is currently visible?" That's more in line with the API docs and avoids implying that the scrollbar is just for use with lists.

Comment (xster) [reply src/_data/catalog/widgets.json]: yup, your wording is better. Changed.

File: src/_data/catalog/widgets.json [hunk 2]
```diff
@@ -586,6 +586,16 @@
...
     "image": "<img alt='' src='/images/catalog-widget-placeholder.png'>"
   },
+  {
+    "name": "CupertinoScrollbar",
+    "description": "An iOS-style scrollbar that indicates which portion of a scrollable widget is currently visible.",
+    "categories": [
+      "Cupertino (iOS-style widgets)"
+    ],
+    "subcategories": [],
+    "link": "https://docs.flutter.io/flutter/cupertino/CupertinoScrollbar-class.html",
```
Comment (sfshaza2) [inline src/_data/catalog/widgets.json]: Please use api.flutter.dev

Comment (xster) [reply src/_data/catalog/widgets.json]: replaced everywhere in the file

**Pull request merged**